src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -101,6 +101,11 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
     private $envCounters = array();
 
     /**
+     * @var int A counter to work around buggy internal name generation for anonymous classes.
```

Comment:
Maybe should you add reference to this bug directly in the source. It will be easier to drop it when the bug will be fixed and we switched to a fixed PHP version (not tomorrow...).